code/modules/research/designs/guncrafting_designs.dm

```diff
@@ -0,0 +1,544 @@
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun
+	category = list("Gun Ammo")
+
+/datum/design/buckshotshell
+	name = "Shotgun Shell: Buckshot"
+	desc = "A buckshot shell."
+	id = "buckshotshell"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/buckshot
+	category = list("Gun Ammo")
+
+/datum/design/rubbershotshell
+	name = "Shotgun Shell: Rubber Shot"
+	desc = "A rubber shot shell."
+	id = "rubbershotshell"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/rubbershot
+	category = list("Gun Ammo")
+
+/datum/design/beanbagslugshell
+	name = "Shotgun Shell: Beanbag Slug"
+	desc = "A beanbag slug shell."
+	id = "beanbagslugshell"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/beanbag
+	category = list("Gun Ammo")
+
+/datum/design/dartshell
+	name = "Shotgun Shell: Dart"
+	desc = "A dart shell."
+	id = "dartshell"
+	req_tech = list("combat" = 1, "materials" = 3)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/dart
+	category = list("Gun Ammo")
+
+/datum/design/dbshell
+	name = "Shotgun Shell: Dragon's Breath"
+	desc = "A dragon's breath shell."
+	id = "dbshell"
+	req_tech = list("combat" = 3, "materials" = 3)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/incendiary/dragonsbreath
+	category = list("Gun Ammo")
+
+/datum/design/fragshell
+	name = "Shotgun Shell: Frag"
+	desc = "A slug shell."
+	id = "fragshell"
+	req_tech = list("combat" = 3, "materials" = 3)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/frag12
+	category = list("Gun Ammo")
+
+/datum/design/breachshell
+	name = "Shotgun Shell: Breaching"
+	desc = "A breaching shell."
+	id = "breachshell"
+	req_tech = list("combat" = 1, "materials" = 3)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/breaching
+	category = list("Gun Ammo")
+
+/datum/design/stunshell
+	name = "Shotgun Shell: Stun"
+	desc = "A stunning shell for a shotgun."
+	id = "stunshell"
+	req_tech = list("combat" = 3, "materials" = 3)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/stunslug
+	category = list("Gun Ammo")
+
+/datum/design/techshell
+	name = "Shotgun Shell: Tech Shell"
+	desc = "A high-tech shotgun shell which can be loaded with materials to produce unique effects."
+	id = "techshotshell"
+	req_tech = list("combat" = 3, "materials" = 3, "powerstorage" = 4, "magnets" = 3)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000, MAT_GLASS = 200)
```

Comment:
Needs a higher material cost. This would be a bit way to low. possibly 1000 silver and 1000 gold aswell. With combat being set to 4.